docs/tutorial/part-four/index.md

```diff
@@ -340,6 +340,11 @@ Another success! 🎉
 
 ![Page title and layout title both pulling from siteMetadata](/site-metadata-two-titles.png)
 
+Why use two different queries here? These examples were quick introductions to
+the query types, how they are formatted, and where they can be used. Just remember,
+only pages can make page queries and non-page components, like Layout, can use
+StaticQuery. In part 7 the tutorial explains these in more depth.
```

Comment:
Can you rewrite this to say: "non-page components. For example, Layout can use StaticQuery. [Part 7](/part-seven/) of the tutorial explains this concept in greater depth."

Reply:
My impression was the whole preceding section is a taster for something to be detailed later thus requiring a _short_ note to point this fact out. @shannonbux that is why I used a simple parenthetical phrase rather than drawing out the topic into its own sentence. 

No worries @amberleyromo I know y'all are trying to get a release out. Your rewrite retains the question which isn't answered by the subsequent sentence. The paragraph probably needs to answer the opening question to be logical.  IMO, the question is necessary because why did the writer use so much space on the query topic if the information wasn't important.

Anecdotal: I recall the first time I walked through this tutorial as a new Gatsby user wondering why I was put through all this work only to throw it away by reverting the work. The balance was off.